execution-engine/contracts/hdac-system/client-api-proxy/src/client_api/mod.rs

```diff
@@ -42,6 +46,8 @@ pub enum Api {
     Delegate(PublicKey, U512),
     Undelegate(PublicKey, Option<U512>),
     Redelegate(PublicKey, PublicKey, U512),
+    Vote(Key, U512),
+    Unvote(Key, U512),
```

Comment:
For api call, I think need to change Unvote.
```
Unvote(Key, Option<U512>)
```

Reply:
67e1226e0c1df3895034547e297d3f7edbeb8b4c